narative/gatsby-theme-novela
# Fixes component shadowing & other things

**Fixes component shadowing for all files.**

For some reason when you used eg. `Article.Hero.tsx` you could not use component shadowing on that file, but when I renamed it to eg. `ArticleHero.tsx` (without period in the middle), then I could use component shadowing. I have tested this and it works now.

To verify:

1. clone this PR & run yarn
2. use component shadowing in `/www` for any component or section
3. run `yarn develop` to verify that it works

---

Other fixes:

- Adds patreon and paypal icons. #92 
- Fixes subscription panel fade when having only one post. #95 
- Fixes Image__Large to not cause horizontal scroll to appear.
- Fixes design bugs that were causing the horizontal scroll to appear on article show. 

Design bug:

![Screenshot_2019-08-27 Getting started with Novela](https://user-images.githubusercontent.com/1839930/63772649-d21bb380-c8d9-11e9-8b4e-0ca5c9250f78.png)

Fixed:

![Screenshot_2019-08-27 Getting started with Novela(1)](https://user-images.githubusercontent.com/1839930/63772656-d6e06780-c8d9-11e9-9f1e-9af7426e946b.png)


Notice the upper black border missing the radius, that is because it went outside the screen and horizontal scroll appeared. That is also solved with this PR.
